internal/grpc/services/storageprovider/storageprovider.go

```diff
@@ -1116,15 +1116,6 @@ func (s *service) RemoveGrant(ctx context.Context, req *provider.RemoveGrantRequ
 	providerID := unwrapProviderID(req.Ref.GetResourceId())
 	defer rewrapProviderID(req.Ref.GetResourceId(), providerID)
 
-	ctx = ctxpkg.ContextSetLockID(ctx, req.LockId)
-
-	// check targetType is valid
-	if req.Grant.Grantee.Type == provider.GranteeType_GRANTEE_TYPE_INVALID {
-		return &provider.RemoveGrantResponse{
-			Status: status.NewInvalid(ctx, "grantee type is invalid"),
-		}, nil
-	}
-
```

Comment:
so ... this is no longer needed?

Reply:
Argh, no I didn't intend to delete this. This probably happened during rebase. I had some conflicts and must have accidentally removed this.